corehq/motech/models.py

```diff
@@ -78,25 +79,25 @@ def get_requests(self, payload_id=None, logger=None):
             auth_type=self.auth_type,
         )
 
-    def is_in_use(self):
+    @property
+    def used_by(self):
```

Comment:
Should this be a cached property?  Looks like it can be referenced multiple times.